program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js

```diff
@@ -0,0 +1,8 @@
+//inverted pyramid
+for(let a = 6;a>=1;a--){
+    let str =' '
+    for(let i = 1;i<=a-1;i++){
+        str+=i + " "
```

Comment:
You should not give 6 and print for 5 pyramids .
It has to print for the given number. 
I have fixed your code to work as expected.

```
for(let a = 5;a>=1;a--){
    let str =' '
    for(let i = 1;i<=a;i++){
        str+=i + " "
    }
    console.log(str)
}
```